src/components/Categories.js

```diff
@@ -1,21 +1,24 @@
-import React from 'react';
+/* eslint-disable */
+
```

Comment:
- [ ] For best practice, kindly make sure you fix your linters errors instead of dasabling them for the entire file.

Reply:
@tresorsawasawa Thanks for the review. I know how important eslint is, but I disabled it because the linter errors only appeared on GitHub and not on my vscode. I didn't know how else to fix it on GitHub since I had already fixed it in my local repo.(That is, my app ran locally without errors of any sort)

Reply:
@blase147 great work implementing some changes requested, however, it's not permitted to disable linters for the entire file kindly try fixing it and also ask for help if you get stuck.

Reply:
HI @blase147 

Sorry to answer you later. it's not permitted to disable linters for the entire file kindly try fixing it and also ask for help if you get stuck.

Once done, you can remove all text about disabling linters